pytorch_lightning/cluster_environments/__init__.py

```diff
@@ -0,0 +1,3 @@
+from pytorch_lightning.cluster_environments.cluster_environment import ClusterEnvironment
```

Comment:
can we rename this package just to `environments` as TE is not a true cluster and there may be another non cluster evns